app/server/me_api/credit_transfer.py

```diff
@@ -282,8 +286,9 @@ def post(self):
                 'feedback': True,
             }
             return make_response(jsonify(response_object)), 400
-
+        incentive_transfers = []
         try:
+            # Question: Do we want the main transfer to go through if the insentive does not? 
```

Comment:
Making a note on this question-- as it stands now hitting the incentive cap does not block the OG transaction from going through. It easily could though!